great_expectations/cli/suite.py

```diff
@@ -338,29 +338,38 @@ def _suite_new(suite: str, directory: str, empty: bool, jupyter: bool, view: boo
 
 
 @suite.command(name="delete")
-@click.option("--suite", "-es", default=None, help="Expectation suite name.")
+@click.argument("suite")
 @click.option(
     "--directory",
     "-d",
     default=None,
-    help="Delete expectation suite.",
+    help="Delete an expectation suite.",
```

Comment:
This isn't the right help string, is it? It should be

```suggestion
    help="The project's great_expectations directory.",
```

For what it's worth, it might be worth refactoring this to:

```python
DIRECTORY_OPTION = click.option("--directory", "-d", default=None, help="...")

@DIRECTORY_OPTION
def cli_command(...):
  pass

@DIRECTORY_OPTION
def another_cli_command(...):
  pass
```

if you're going to reuse `DIRECTORY_OPTION` more than a few times (which is looking likely) and helps avoid inconsistent help strings.

Can be done in a separate PR though :)